public/favicon/manifest.json

```diff
@@ -0,0 +1,41 @@
+{
+ "name": "App",
```

Comment:
fix: Lets not include public/ into reviews. So we have to add this to `.gitignore`